src/cryptonote_core/cryptonote_core.cpp

```diff
@@ -1725,7 +1725,7 @@ namespace cryptonote
       m_starter_message_showed = true;
     }
 
-    m_txpool_auto_relayer.do_call(boost::bind(&core::relay_txpool_transactions, this));
+    relay_txpool_transactions(); // txpool handles periodic DB checking
```

Comment:
There's a `m_mempool.on_idle()` call below I didn't hjijack, but perhaps should've Pointing this out for reviewers.